src/ComponentCover/index.js

```diff
@@ -0,0 +1,32 @@
+import React, { Fragment } from 'react'
+import propTypes from 'prop-types'
+import css from 'styled-jsx/css'
+import cx from 'classnames'
+
+const ComponentCover = ({ children }) => (
+    <div className="component-cover">
+        {children}
+        <style jsx>{`
+            div {
+                display: flex;
+                align-items: center;
+                justify-content: center;
+
+                position: relative;
+                left: 0;
+                top: 0;
+
+                height: 100%;
+                width: 100%;
+
```

Comment:
I think we should add a z-index here too...

Reply:
c341aec